frontend/src/components/layout/Header/Partials/CustomSelect/CustomSelect.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import Select, { PropsValue } from 'react-select';
+import { pxToRem, colors } from '../../../../../styled-theme';
+import { SelectOptions } from '../partials.types';
+
+export interface CustomSelectProps {
+  readonly isMenuOpen: boolean;
+  readonly setIsMenuOpen: (value: React.SetStateAction<boolean>) => void;
```

Comment:
Could this just be state within this component?

Reply:
For sure!  Done.

Reply:
I don't think you've pushed your changes for this yet

Reply:
Awesome it's there now